src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/ApprenticeController.cs

```diff
@@ -354,13 +355,68 @@ public async Task<IActionResult> EditApprenticeship(EditApprenticeshipRequest re
         [Authorize(Policy = nameof(PolicyNames.HasAccountOwnerPermission))]
         public async Task<IActionResult> EditApprenticeship(EditApprenticeshipRequestViewModel viewModel)
         {
+            var apprenticeship = await _commitmentsApiClient.GetApprenticeship(viewModel.ApprenticeshipId);
+
+            if (viewModel.CourseCode != apprenticeship.CourseCode || 
+                (viewModel.CourseCode == apprenticeship.CourseCode && apprenticeship.StartDate <= viewModel.StartDate.Date.Value))
```

Comment:
Can tweak this. Either the viewModel.CourseCode != apprenticeship.CourseCode is true and it enters the if, or it false and it switches to the right hand side not short circuiting.

Therefore, you know already to be on the right hand side, viewmodel.CourseCode == apprenticeship.CourseCode so it's not needed.

On the Employer side there is:
```if (viewModel.CourseCode != apprenticeship.CourseCode || (apprenticeship.StartDate != viewModel.StartDate.Date.Value && apprenticeship.StartDate < viewModel.StartDate.Date.Value))```

but I think i'll change it to 

```if (viewModel.CourseCode != apprenticeship.CourseCode || apprenticeship.StartDate < viewModel.StartDate.Date.Value))```

As if the apprenticeship.StartDate != to the view model start date, you know they are not equal if it satisfies the latter condition ( specifically without the = ), if it's the same date, you don't want to calculate either. So i'd suggest the same for the above.

Reply:
Updated to `if (viewModel.CourseCode != apprenticeship.CourseCode || apprenticeship.StartDate < viewModel.StartDate.Date.Value))`